app/models/gql/Arguments.scala

```diff
@@ -63,6 +64,27 @@ object Arguments {
     "Use disease ontology to capture evidences from all descendants to build associations"
   )
 
+  val ComparerEnum = EnumType(
```

Comment:
You can use a `deriveEnumType` macro here if you wanted to.
See: https://sangria-graphql.github.io/learn/#enumtype-derivation

Reply:
Changed to derived enum